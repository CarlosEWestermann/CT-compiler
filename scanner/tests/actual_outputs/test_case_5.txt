syntax error, unexpected TK_LIT_INT, expecting TK_PR_INT or TK_PR_FLOAT or TK_PR_BOOL or ')'