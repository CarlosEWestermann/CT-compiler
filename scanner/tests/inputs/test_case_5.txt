if(valor_1 == 10 || valor_2 != 5) {
    return false;
}
