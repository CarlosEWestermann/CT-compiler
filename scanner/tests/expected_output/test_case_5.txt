1 TK_PR_IF [if]
1 TK_ESPECIAL [(]
1 TK_IDENTIFICADOR [valor_1]
1 TK_OC_EQ [==]
1 TK_LIT_INT [10]
1 TK_OC_OR [|]
1 TK_OC_OR [|]
1 TK_IDENTIFICADOR [valor_2]
1 TK_OC_NE [!=]
1 TK_LIT_INT [5]
1 TK_ESPECIAL [)]
1 TK_ESPECIAL [{]
2 TK_PR_RETURN [return]
2 TK_LIT_FALSE [false]
2 TK_ESPECIAL [;]
3 TK_ESPECIAL [}]
